Example/Controllers/DisplayController.swift

```diff
@@ -1,23 +1,27 @@
 import UIKit
 import MiniApp
+import AVKit
```

Comment:
the use of AVKit is really annoying to me. I know it's really convenient and maybe the only solution, but it's an heavy framework to just manage orientation

Reply:
Sadly yes, I don't like it either.
And if we don't use this, then the orientation lock is not working for the video player. Funny thing is, even if the app supports only portrait mode, the video player can rotate to all orientations(without this extension). So having this code in place helps us to lock the orientation.